target/linux/mediatek/image/mt7629.mk

```diff
@@ -7,3 +7,13 @@ define Device/mediatek_mt7629-rfb
   DEVICE_PACKAGES := swconfig
 endef
 TARGET_DEVICES += mediatek_mt7629-rfb
+
+define Device/iptime_a6004mx
+  DEVICE_VENDOR := ipTIME
+  DEVICE_MODEL := A6004MX
+  DEVICE_DTS := mt7629-iptime-a6004mx
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_PACKAGES := kmod-usb3
+  SUPPORTED_DEVICES := iptime,a6004mx
```

Comment:
```iptime,a6004mx``` is included in ```SUPPORTED_DEVICES``` by default, this line can be dropped. see [here](https://github.com/openwrt/openwrt/blob/5205010a546fed2227c208f6bd025b47398f4cf3/include/image.mk#L408)

Reply:
resolved